src/engine/loopingcontrol.cpp

```diff
@@ -126,6 +126,8 @@ LoopingControl::LoopingControl(QString group,
                                           true, false, false, 4.0);
     m_pCOBeatLoopSize->connectValueChangeRequest(this,
             SLOT(slotBeatLoopSizeChangeRequest(double)), Qt::DirectConnection);
+    m_pCOBeatLoopSizeIndicator = new ControlObject(ConfigKey(group, "beatloop_size_indicator"));
```

Comment:
Unless you can suggest an alternative that requires none of the below:
  * imperfect rounding
  * forbidding manually entering arbitrary loop sizes to the spinbox
  * reaching for the keyboard and mouse to go back to exact integer beatloop sizes after setting a manual loop

Then I think this is the right way to go.